### ISSUE TITLE
Users need access to the NYC tileset

### ISSUE THREAD

**Issue by hpinkos** [CesiumGS/cesium-workshop#10]
The workshop includes instructions for signing up for an ion account, but then they have no way to access the NYC tileset because it is not in the asset depot.

@mramato is this something we can make available for users to add to their accounts?

**Comment by mramato** [comment]
Absolutely, I honestly don't know why we've waited so long to do it.  We just need to be sure to use the latest version of the data set and get it clamped to CWT.

**Comment by shehzan10** [comment]
Is there any interest in generate 2 tilesets - 1 White (as currently exists) and 1 colored - like the "usual" citygml data representation with red roofs etc?

**Comment by mramato** [comment]
I don't think we want to put 2 copies of NYC in the Asset Depot, so a single copy is preferred, clamped to CWT.  We should decide if there is specific tile properties/metadata we want to include.  For example we could change the workshop to color all of the roofs red based on metadata?  Might be too much for a first iteration, just throwing ideas out there.

**Comment by mramato** [comment]
But my default answer is, white tileset clamped to CWT and that's it.

**Comment by hpinkos** [comment]
Also reported by @jonathanvhive in https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues/242

**Comment by luipir** [comment]
AssetId changed from 3839 to 75343
var city = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(75343) }));

and heightOffset respect ellipsoid would be set to 0 instead of -32

**Comment by luipir** [comment]
there are also some minor error in the App.js... do you want a PR to fix them?

**Comment by luipir** [comment]
https://github.com/CesiumGS/cesium-workshop/pull/18

**Comment by luipir** [comment]
don a PR to fix access to the asset... that have to be added into MyAssets
https://github.com/CesiumGS/cesium-workshop/pull/18
